forum/models/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""
+Mongo Models
+"""
+
+from .comments import Comment
+from .contents import Contents
+from .threads import CommentThread
+from .users import Users
```

Comment:
we can add at the end.
```
__all__ = [
    "Comment",
    "Contents",
    "CommentThread",
    "Users",
]
```
so that we can import models like
`from forum.models import Contents` instead of `from forum.models.contents import Contents`
same goes for other imports for models,
same can be done for serializers and view or at least serializers.